### PR TITLE
BibFormat: export bibdocfiles in FFT

### LINKED ISSUE #206: Obsolete ICNs are wrongly mapped in XME

### PR DESCRIPTION
**Pull request by kaplun**
* Also fix bug in exporting obsolete ICNs.
  (closes #206)

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

**Comment by kaplun** [comment]
cc @jacquerie

**Comment by jacquerie** [comment]
Is this deployed?

**Comment by kaplun** [comment]
Yup! But you can find FFTs only with new data.

File: bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py [hunk 1]
```diff
@@ -211,6 +232,27 @@ def format_element(bfo, oai=0):
                     if len(ids) == 1:
                         subfields.append(('z', '%i' % ids[0]))
 
+    for field in record_get_field_instances(record, '110'):
+        subfields = field_get_subfield_instances(field)
+        subfield_dict = dict(subfields)
+        if is_institution:
+            # We try to resolve obsolete ICNs
+            if 'w' in subfield_dict:
```
Comment (jacquerie) [inline bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py]: Shouldn't this guy be `x`? https://twiki.cern.ch/twiki/bin/view/Inspire/DevelopmentRecordMarkupInstitutions#Field_Mapping_final

Comment (kaplun) [inline bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py]: F..k! :scream:

Comment (kaplun) [inline bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py]: Fixed in #212

**Pull request merged**